packages/frontend/src/server/features/utils/get-badge-link.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Badge, Project } from '@l2beat/config'
+import { assertUnreachable } from '@l2beat/shared-pure'
+import { getFilterSearchParams } from '~/components/table/filters/utils/get-filter-search-params'
+
+export function getBadgeLink(
```

Comment:
This function would be easier to implement if this info was stored in badges properly.

You can have the following config in a badge
```
action?: {
   scalingFilter?: [key, value]
   publicDaHighlight?: string
   selfDaHightlight?: boolean
}
```

Reply:
Or maybe even as a discriminated union